src/docs/reference/core/Modules/README.md

```diff
@@ -60,9 +60,12 @@ You can find a sample application here: [`OrchardCore.Mvc.Web`](../../../../Orch
 
 The following configuration values are used by default and can be customized:
 
+By default each tenant doesn't serve static files from Host `wwwroot`. To serve Host `wwwroot` for each tenant, set `ServeHostWebRoot` to `true` in tenant config.
```

Comment:
Has it always been the case?

Reply:
Currently  static files from Host `wwwroot` is only served to `Default` tenant  (and not to other tenants) by configuring `app.UseStaticFiles()` in Host startup.

Keeping the same behavior by setting default value to `false`